contao/dca/tl_metamodel_rendersetting.php

```diff
@@ -65,13 +66,14 @@
 
 $GLOBALS['TL_DCA']['tl_metamodel_rendersetting']['fields']['file_imageSize'] = array
 (
-    'label'                   => &$GLOBALS['TL_LANG']['tl_metamodel_rendersetting']['file_imageSize'],
+    'label'                   => &$GLOBALS['TL_LANG']['tl_metamodel_rendersetting']['file_imageSizef'],
     'exclude'                 => true,
     'inputType'               => 'imageSize',
-    'options'                 => $GLOBALS['TL_CROP'],
+    'options'                 => \Contao\System::getImageSizes(),
```

Comment:
Can we make this an `option_callback` to ease migration of the feature to Contao 4 please (we want a service there)?

Reply:
Yeah, that's a good idea. Solve this with event.

Reply:
@baumannsven ready?

Reply:
The image size options comes yet over the GetPropertyOptionsEvent